en/docs/guides/identity-federation/configure-ws-trust.md

```diff
@@ -46,141 +41,107 @@ To secure the Security Token Service:
-					  <br/>
-				   </div>
-				</div>
-			 </td>
-			 <td><code>               https://localhost:9444/services/echo              </code></td>
-		  </tr>
-		  <tr class="even">
-			 <td><strong>Certificate Alias</strong></td>
-			 <td>This is the alias of the certificate.</td>
-			 <td><code>               wso2carbon              </code></td>
-		  </tr>
-	   </tbody>
-	</table>
-    
-	![add-new-trusted-service]({{base_path}}/assets/img/tutorials/add-new-trusted-service.png)
+## Configure the service provider
+
+The steps in this section should be followed if you are using Holder of Key as the [subject confirmation method]({{base_path}}/references/concepts/authentication/intro-ws-trust/#subject-confirmation).
+
+<!-- For more information, see [how to configure STS for obtain tokens with Holder-Of-Key subject confirmation]({{base_path}}/learn/configuring-sts-for-obtaining-tokens-with-holder-of-key-subject-confirmation). -->
+
+!!! Tip "Before you begin"
+    You must first [register a service provider]({{base_path}}/guides/applications/register-sp/).
+
+1. On the Management console, go to **Service Providers** > **List** and select your service provider.
+2. Expand **Inbound Authentication Configuration** > **WS-Trust Security Token Service Configuration** and click **Configure**.
+3. Enter the required details as given below.
+    <table>
+        <thead>
+            <tr class="header">
+                <th>Field</th>
+                <th>Description</th>
+                <th>Sample Value</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr class="odd">
+                <td><strong>Endpoint Address</strong></td>
+                <td>
+                    <div class="content-wrapper">
+                    <div>
+                        Enter the trusted relying party's <strong>endpoint address,</strong> which is the endpoint address of the Security Token Service.
+                        <p>The endpoint must be used as the service URL to which the token gets delivered by the STS client. Then select the public certificate imported. Tokens issued are encrypted using the public certificate of the trusted relying party. Therefore, the consumer who obtains this token to invoke the RP service will not be able to see the token.</p>
+                        <div class="admonition note">
+                            <p class="admonition-title">Note</p>
+                            Make sure to upload the certificate of the relying party to the trust store. For instructions, see <a href="https://docs.wso2.com/display/ADMIN44x/Creating+New+Keystores#CreatingNewKeystores-ca_certificateAddingCA-signedcertificatestokeystores">adding CA-signed certificates to keystores</a>.
```

Comment:
This should be the link: https://is.docs.wso2.com/en/latest/deploy/security/create-new-keystores/#add-ca-signed-certificates-to-keystores